core/services/job/orm.go

```diff
@@ -55,6 +55,9 @@ type ORM interface {
 	FindPipelineRunIDsByJobID(jobID int32, offset, limit int) (ids []int64, err error)
 	FindPipelineRunsByIDs(ids []int64) (runs []pipeline.Run, err error)
 	CountPipelineRunsByJobID(jobID int32) (count int32, err error)
+
+	JobsByPipelineSpecIDs(ids []int32) ([]Job, error)
```

Comment:
Can we rename this please 

```suggestion
	FindJobsByPipelineSpecIDs(ids []int32) ([]Job, error)
```